pkg/apis/events/v1alpha1/scheduler_types.go

```diff
@@ -108,6 +114,11 @@ func (s *Scheduler) ConditionSet() *apis.ConditionSet {
 	return &schedulerCondSet
 }
 
+// SchedulerEventSource returns the Cloud Scheduler CloudEvent source value.
+func SchedulerEventSource(jobName string) string {
+	return fmt.Sprintf("//cloudscheduler.googleapis.com/%s", jobName)
```

Comment:
I'm wondering maybe I should use "//cloudscheduler.googleapis.com/v1beta1/" instead of "//cloudscheduler.googleapis.com/".  FYI: https://cloud.google.com/scheduler/docs/reference/rest/

Reply:
I'm more inclined to not include `v1beta1`. Let's leave it as is for now, and then we may revisit